templates/src/app.js

```diff
@@ -1,31 +1,60 @@
-var dotenv = require("dotenv");
-var express = require("express");
-var morgan = require("morgan");
+const dotenv = require("dotenv");
+const express = require("express");
+const morgan = require("morgan");
+const dbDatasource = require("./db");
+<%if eq (index .Params `apiType`) "graphql" %>const { ApolloServer } = require("apollo-server-express");
 
-var { connect } = require("./db");
+const typeDefs = require("./app/graphql/schema");
+const { createStore } = require("./app/graphql/initdb");
+const resolvers = require("./app/graphql/resolvers");
+const LaunchAPI = require("./app/graphql/datasources/LaunchAPI");
+const UserDB = require("./app/graphql/datasources/UserDB");
+const FileAPI = require("./app/graphql/datasources/FileAPI");
+const AuthAPI = require("./app/graphql/datasources/AuthAPI");<% end %>
+<%if eq (index .Params `fileUploads`) "yes" %>const fileRoutes = require("./app/file");<% end %>
 const statusRoutes = require("./app/status");
-<%if eq (index .Params `fileUploads`) "yes" %>const fileRoutes = require("./app/file");
-<% end %><%if eq (index .Params `userAuth`) "yes" %>const authRoutes = require("./app/auth");
-<% end %>
+<%if eq (index .Params `userAuth`) "yes" %>const authRoutes = require("./app/auth");<% end %>
+
 dotenv.config();
-var app = express();
+const app = express();
 app.use(morgan("combined"));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+<%if eq (index .Params `apiType`) "graphql" %>
+const store = createStore();
+
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+  dataSources: () => ({
+    launchAPI: new LaunchAPI(),
```

Comment:
Can we just get rid of the launch stuff? Seems like it's not really adding much value since the other stuff give some reasonable examples of implementing resolvers.

Reply:
Removed LaunchAPI. Thanks.